src/test/emulators/auth/misc.spec.ts

```diff
@@ -39,6 +44,31 @@ describeAuthEmulator("token refresh", ({ authApi }) => {
       });
   });
 
+  it("should populate auth_time to match lastLoginAt (in seconds since epoch)", async () => {
+    const emailUser = { email: "alice@example.com", password: "notasecret" };
+    const { refreshToken } = await registerUser(authApi(), emailUser);
+
+    const res = await authApi()
```

Comment:
Please advance the mocked clock by a few seconds before this so that we can verify that the auth_time is indeed the login time, not the refresh time. You may use this as an inspiration:

https://github.com/firebase/firebase-tools/blob/66447909ab6f2defd04b257b48193c70d778c7ea/src/test/emulators/auth/oob.spec.ts#L215